wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -38,8 +38,9 @@
             </div>
 
             <div id="nav-toggle" class="nav-toggle icon text-replace">{% trans "Menu" %}</div>
-
-            {% block content %}{% endblock %}
+            <main>
```

Comment:
It is [still advised](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#Browser_compatibility) to include `role="main"` for IE11 which is still a supported browser. Probably worth including :)

Reply:
Yes, this should still be a `div` tag with the appropriate role to maintain browser support without extra JS.